tests/python/unittest/test_operator.py

```diff
@@ -5237,6 +5238,17 @@ def custom_add():
         p.join(5)
         assert not p.is_alive(), "deadlock may exist in custom operator"
 
+    # test except handling
+    # see https://github.com/apache/incubator-mxnet/pull/14575
+    def custom_add_exc():
```

Comment:
Can we add a comment that an exception is expected due to shapes I assume?

Reply:
Sure, I'll add it.